docker/build/Dockerfile.unprivileged.alma9

```diff
@@ -0,0 +1,133 @@
+# This dockerfile assumes that you're passing the binary with its corresponding arguments to run it.
```

Comment:
A container that can be run this way is nice to have, but I think it's better to modify our current container to allow it rather than create a different install procedure, as this one is a lot more complicated than what we currently have, and can become out of date, as dependencies, build options, etc, are hard-coded. For example, if we remove ENABLE_SCITOKENS option in the future, it would require changing this file. I also think that it is already possible to do this with our current container if you pass the full command line with `xrootd ...` when running the container, no?